index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
This code establishes size and color for the Tetris blocks, the changes are good for making each block distinct from the other.